src/Keboola/Google/BigQuery/RestApi/Client.php

```diff
@@ -123,6 +124,86 @@ public function listProjects()
         return $return;
     }
 
+    public function datasetExists(string $projectId, string $datasetId): bool
+    {
+        try {
+            $url = sprintf(
+                'https://www.googleapis.com/bigquery/v2/projects/%s/datasets/%s',
+                $projectId,
+                $datasetId
+            );
+
+            $response = $this->request($url, 'GET');
+
+            $responseBody = \GuzzleHttp\json_decode($response->getBody(), true);
+
+            if ($response->getStatusCode() == 200 && !empty($responseBody['selfLink'])) {
```

Comment:
toto by som asi napisal ako `return $response->getStatusCode() == 200 && !empty($responseBody['selfLink'])` ... aj ked neviem ake ine pripady tam mozu nastat, no.

Reply:
ide mi o to, ze ked mas tym navratovej hodnoty `bool`, tak nech je to pravda.

Reply:
Jo to mas pravdu